src/main/java/org/minions/devfund/damianvp/battleship/Ocean.java

```diff
@@ -0,0 +1,189 @@
+package org.minions.devfund.damianvp.battleship;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Random;
+
+/**
+ * Ocean class.
+ */
+public class Ocean {
+    static final int LENGTH = 20;
+    static final int TOTAL_SHIPS = 13;
+    private Ship[][] ships = new Ship[LENGTH][LENGTH];
```

Comment:
This should be instantiate in the constructor

Reply:
ok, I will move to constructor